ROMFS/px4fmu_common/init.d/airframes/13015_jiyi_batman

```diff
@@ -0,0 +1,96 @@
+#!/bin/sh
+#
+# @name JIYIRobot Batman
+#
+# @type VTOL Tiltrotor
+# @class VTOL
+#
+# @maintainer Stone White <stone@thone.io>
+# @creator Yu Liu <lydoraemon@gmail.com>
+#
+# @output MAIN1 Motor right
+# @output MAIN2 Motor left
+# @output MAIN3 Motor back
+# @output MAIN4 empty
+# @output MAIN5 Tilt servo right
+# @output MAIN6 Tilt servo left
+# @output MAIN7 Elevon right
+# @output MAIN8 Elevon left
+#
+# @board intel_aerofc-v1 exclude
+# @board bitcraze_crazyflie exclude
+#
+
+sh /etc/init.d/rc.vtol_defaults
+
+if [ $AUTOCNF = yes ]
+then
+	param set CBRK_AIRSPD_CHK 162128
+
+	param set FW_ARSP_MODE 2
+
```

Comment:
I know you copy some from 13012.I personally think that there is no need for blank lines here.

Reply:
Fixed